python/ray/tune/tune.py

```diff
@@ -1066,14 +1066,21 @@ class and registered trainables.
             mode=mode,
         )
     else:
+        from ray.tune.experimental.output import AirResultProgressCallback
+
         air_progress_reporter = _detect_air_reporter(
             air_verbosity,
             num_samples=search_alg.total_samples,
             entrypoint=_entrypoint,
             metric=metric,
             mode=mode,
             config=config,
+            progress_metrics=progress_metrics,
         )
+        for callback in callbacks:
+            if isinstance(callback, AirResultProgressCallback):
```

Comment:
Can we directly create the `AirResultProgressCallback` with progress reporter in `_create_default_callbacks`?

Reply:
I see. That makes sense. The apis will be simplified after we have a context singleton.